### PR TITLE
Moves PowerFeed Utilization Bar to Standard Template

### PR DESCRIPTION
**Pull request by jvanderaa**
### Fixes: #308 

- Updates the template tag rendered for the utilization_graph_raw_data from one that does not exist to the existing one.

### CODE REVIEW

**Comment by glennmatthews** [comment]
Thanks for the PR! This does look like the correct fix, but I'd really like to understand why our existing view test automation didn't catch this issue.

**Comment by jathanism** [comment]
> Thanks for the PR! This does look like the correct fix, but I'd really like to understand why our existing view test automation didn't catch this issue.

It's because this would only be caught when rendering the view. There are no tests for this that I can determine, but there should be.

**Comment by glennmatthews** [comment]
> > Thanks for the PR! This does look like the correct fix, but I'd really like to understand why our existing view test automation didn't catch this issue.
> 
> It's because this would only be caught when rendering the view. There are no tests for this that I can determine, but there should be.

This was bugging me, so I took a closer look.

- `PowerFeedTestCase` is a subclass of `ViewTestCases.PrimaryObjectViewTestCase`.
- `ViewTestCases.PrimaryObjectViewTestCase` includes `GetObjectViewTestCase`.
- `GetObjectViewTestCase` includes the test `test_get_object_with_permission`
- `test_get_object_with_permission` performs `self.assertHttpStatus(self.client.get(instance.get_absolute_url()), 200)`.

So while we *do* have a test case for this view, it looks like this test wasn't encountering #308 because the template rendering issue in #308 is only encountered when `object.connected_endpoint.get_power_draw` exists - and the baseline data created in `PowerFeedTestCase.setUpTestData` creates several `PowerFeed` objects but doesn't actually connect anything to them. By contrast the new test case that @jvanderaa has added does set up such a connection, and so should catch any future regression of this issue. :-)

File: nautobot/dcim/tests/test_views.py [hunk 1]
```diff
@@ -1916,23 +1916,33 @@ def setUpTestData(cls):
...
             Rack.objects.create(site=site, name="Rack 1"),
             Rack.objects.create(site=site, name="Rack 2"),
         )
 
-        PowerFeed.objects.create(name="Power Feed 1", power_panel=powerpanels[0], rack=racks[0])
-        PowerFeed.objects.create(name="Power Feed 2", power_panel=powerpanels[0], rack=racks[0])
-        PowerFeed.objects.create(name="Power Feed 3", power_panel=powerpanels[0], rack=racks[0])
+        powerfeed_1 = PowerFeed.objects.create(name="Power Feed 1", power_panel=powerpanels[0], rack=racks[0])
+        powerfeed_2 = PowerFeed.objects.create(name="Power Feed 2", power_panel=powerpanels[0], rack=racks[0])
+        powerfeed_3 = PowerFeed.objects.create(name="Power Feed 3", power_panel=powerpanels[0], rack=racks[0])
```
Comment (glennmatthews) [inline nautobot/dcim/tests/test_views.py:1937]: Travis is failing because `powerfeed_3` is an unused variable.

**Review by glennmatthews — APPROVED** [review]
Nice work!

File: nautobot/dcim/tests/test_views.py [hunk 2]
```diff
@@ -1969,3 +1979,26 @@ def setUpTestData(cls):
...
+            name="Device1",
+            site=self.site,
+        )
+
+        powerport = PowerPort.objects.create(device=device, name="Power Port 1")
+
+        powerfeed = self.powerfeeds[0]
+
+        cable = Cable(termination_a=powerport, termination_b=powerfeed, status=Status.objects.get(slug="connected"))
+        cable.save()
```
Comment (jathanism) [inline nautobot/dcim/tests/test_views.py:2001]: Can you please try changing this to `Cable.objects.create` and ditch the next line and see if it still works? Just for consistency, thanks.

```suggestion
        cable = Cable.objects.create(termination_a=powerport, termination_b=powerfeed, status=Status.objects.get(slug="connected"))
```

Comment (jvanderaa) [reply nautobot/dcim/tests/test_views.py]: @jathanism this works as well. I have pushed the update into the repo.

**Review by jathanism — APPROVED** [review]

**Pull request merged**